e2e-tests/pitr-physical/conf/some-name-rs0.yml

```diff
@@ -54,11 +54,9 @@ spec:
       antiAffinityTopologyKey: none
     resources:
       limits:
-        cpu: 500m
-        memory: 1G
+        memory: 4G
       requests:
-        cpu: 100m
-        memory: 0.1G
+        memory: 1G
```

Comment:
@hors @tplavcic I'm not sure if we should change these but Dmytro pointed out that our previous memory limits are simply not enough to run physical restore.